setup.py

```diff
@@ -42,20 +46,41 @@ def maybe_download_cub():
     return [os.path.abspath("./include")]
 
 
-def cuda_extension():
+def cuda_extension(fast_build=False):
     import torch
     from torch.utils.cpp_extension import CUDAExtension
+    SMs = None
+    if fast_build:
+        SMs = []
+        devices = torch.cuda.device_count()
+        print('FAST_BUILD_CUAEV: ON')
+        print('This build will only support the following devices or the devices with same cuda capability: ')
+        for i in range(devices):
+            d = 'cuda:{}'.format(i)
+            sm = torch.cuda.get_device_capability(i)
+            sm = int(f'{sm[0]}{sm[1]}')
+            if sm >= 50:
```

Comment:
Why SM>50? If you have a machine with a SM<50 GPU, why not build on it?

Reply:
this is to disable for some common graphics card like sm35..